packages/transaction-controller/src/TransactionController.ts

```diff
@@ -2676,4 +2677,25 @@ export class TransactionController extends BaseControllerV1<
       log('Error while updating post transaction balance', error);
     }
   }
+
+  private async publishTransactionForRetry(
+    rawTx: string,
+    transactionMeta: TransactionMeta,
+  ): Promise<string> {
+    let hash;
```

Comment:
Rather than using a `let` here, could we instead just use a `const` and `return` it inside the `try` block?